db/seeds.rb

```
@@ -5,3 +5,6 @@
 #
 #   movies = Movie.create([{ name: 'Star Wars' }, { name: 'Lord of the Rings' }])
 #   Character.create(name: 'Luke', movie: movies.first)
+#
+
+ImageLink.create([{ link: 'http://i.imgur.com/PVrX7wZ.jpg' }])
```

What will happen if this create fails? How can we be sure that such a failure isn't silently dropped?